packages/expo-cli/lib/configure-plugin.js

```diff
@@ -1,9 +1,28 @@
 const { join } = require('path')
 const { readFile, writeFile } = require('fs')
 const { promisify } = require('util')
+const { spawn } = require('child_process')
+const { blue } = require('kleur')
 
 const plugin = '@bugsnag/plugin-expo-eas-sourcemaps'
 
+function usingWorkspaces (projectRoot) {
+  return new Promise((resolve, reject) => {
+    const command = ['workspaces', 'info']
```

Comment:
When I run `yarn workspaces info` on `yarn 3.2.2` I get command not found:

```
Unknown Syntax Error: Command not found; did you mean:

$ yarn workspaces list [--since] [-R,--recursive] [-v,--verbose] [--json]
While running workspaces info
```

Maybe this is because it's not a workspace but then it proceeds to list available commands (just list), which makes me think it would be listed if it exists in this version of yarn. Even after `yarn plugin import workspace-tools` I don't see the info command:

```
Unknown Syntax Error: Command not found; did you mean one of:

  0. yarn workspaces list [--since] [-R,--recursive] [-v,--verbose] [--json]
  1. yarn workspaces focus [--json] [--production] [-A,--all] ...
  2. yarn workspaces foreach [-R,--recursive] [--from #0] [-A,--all] [-v,--verbose] [-p,--parallel] [-i,--interlaced] [-j,--jobs #0] [-t,--topological] [--topological-dev] [--include #0] [--exclude #0] [--no-private] [--since] <commandName> ...

While running workspaces info
```

I can't find something definitive saying it was removed in yarn 2+ but it's not listed as a command here https://yarnpkg.com/cli/workspaces/list. 

So I'm wondering if this approach will only work for yarn 1.x?

Reply:
ahh. that could be possible. I'll check if list covers what we're after

Reply:
Also `yarn workspaces list --json`

gives me 

```
yarn workspaces list --json
{"location":".","name":"dashboard"}
```
even though the repo I ran it is is not a workspace.

Reply:
how useful! okay, I'll have to consider another option - thanks!